common/predictive-text/message.d.ts

```diff
@@ -41,8 +41,8 @@ type Token = number;
 /**
  * The valid outgoing message kinds.
  */
-type OutgoingMessageKind = 'ready' | 'suggestions';
-type OutgoingMessage = ReadyMessage | SuggestionMessage;
+type OutgoingMessageKind = 'ready' | 'suggestions' | 'word';
+type OutgoingMessage = ReadyMessage | SuggestionMessage | BrokenWordMessage;
```

Comment:
Since `ReadyMessage` uses `'ready` as its message type, and `SuggestionMessage` uses `*'suggestions'` as its message type, I'm wondering if we can do the same with `BrokenWordMessage` (and `SuggestionsMessage` for that matter! 😂)

For example,

```suggestion
type OutgoingMessage = ReadyMessage | SuggestionMessage | WordMessage;
```

```suggestion
type OutgoingMessage = ReadyMessage | SuggestionMessage | CurrentWordMessage;
```

...but that would incur changing the message type itself to `currentword` :/

Reply:
Sounds reasonable.  I debated the names internally quite a bit, with nothing feeling quite right.  Most of the names I wanted to choose were going to involve reusing the same message name for both directions, which definitely didn't feel right given the rest of our existing design.